packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -184,21 +184,22 @@ const DialogueRecommendationRow = ({ rowProps, classes, showSuggestedTopics }: D
 
   if (!currentUser) return <></>;
   const preTopicRecommendations: TopicRecommendationWithContents[] | undefined = topicData?.GetTwoUserTopicRecommendations; 
-  const topicRecommendations = preTopicRecommendations?.filter(topic => topic.theirVote !== null);
+  const topicRecommendations = preTopicRecommendations?.filter(topic => ['agree', 'disagree'].includes(topic.theirVote) ); // todo: might want better type checking here in future for values of theirVote
```

Comment:
TODO might be addressed by using a `TupleSet`; not sure what the exact concern is